Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/ConceptTensorFlowObjectDetectionWebcam.java

```diff
@@ -84,9 +40,11 @@ public class ConceptTensorFlowObjectDetection extends LinearOpMode {
      * and paste it in to your code on the next line, between the double quotes.
      */
     private static final String VUFORIA_KEY =
-            //" -- YOUR NEW VUFORIA KEY GOES HERE  --- ";
-    // Vuforia licence key - Truong
```

Comment:
Um did you move the one from TeamMentor here? Is it still there? You shouldn't delete the one in TeamMentor.

Reply:
It looks like you deleted the one that @tmicro made and created your own so github thinks you moved it because it has the same name.